linker/Mono.Linker.Steps/AddBypassNGenStep.cs

```diff
@@ -0,0 +1,121 @@
+﻿//
+// AddBypassNGenStep.cs
+//
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Mono.Cecil;
+using Mono.Collections.Generic;
+using Mono.Cecil.Cil;
+
+namespace Mono.Linker.Steps {
+
+	public class AddBypassNGenStep : BaseStep{
+
+		AssemblyDefinition coreLibAssembly;
+		CustomAttribute bypassNGenAttribute;
+		
+		protected override void ProcessAssembly (AssemblyDefinition assembly)
+		{
+			if (Annotations.GetAction (assembly) == AssemblyAction.AddBypassNGen) {
+				coreLibAssembly = Context.Resolve (assembly.MainModule.TypeSystem.CoreLibrary);
+				bypassNGenAttribute = null;
```

Comment:
This is still quite confusing. Would be much better to split GetBypassNGenAttribute into 2 methods (e.g. DefineBypassNGenAttribute and GetBypassNGenAttribute) and call DefineBypassNGenAttribute one only here ?

Reply:
BypassNGenAttribute may already be defined in corelib so we may not have to define it and the name DefineBypassNGenAttribute would be confusing.

I changed the name of the method to EnsureByapssNGenAttribute and made its return type void. Do you like this version better?